amd64.env

```
@@ -1,7 +1,7 @@
 rosdistro=galactic
 rmw_implementation=rmw_cyclonedds_cpp
 base_image=ubuntu:20.04
-cuda_base_image=nvidia/cuda:11.4.3-devel-ubuntu20.04
-cuda_version=11.4
-cudnn_version=8.2.4.15-1+cuda11.4
-tensorrt_version=8.2.4-1+cuda11.4
+cuda_base_image=nvidia/cuda:11.6.2-devel-ubuntu20.04
+cuda_version=11.6
+cudnn_version=8.4.1.50-1+cuda11.6
+tensorrt_version=8.4.1.5-1+cuda11.6
```

Probably we need a new version? :thinking: 
https://github.com/autowarefoundation/autoware/pull/2766#issuecomment-1203720547

```console
❯ find-cuda-versions --os ubuntu2004 --cuda 11.6
[cudnn x86_64]
8.4.0.27-1+cuda11.6
8.4.1.50-1+cuda11.6

[cudnn sbsa]
8.4.0.27-1+cuda11.6
8.4.1.50-1+cuda11.6

[tensorrt x86_64]
8.4.1.5-1+cuda11.6
8.4.2.4-1+cuda11.6

[tensorrt sbsa]
8.4.1.5-1+cuda11.6
8.4.2.4-1+cuda11.6
```

```suggestion
tensorrt_version=8.4.2.4-1+cuda11.6
```

---

It seems so. @wep21 Could you confirm this change? :pray: